jovian/jovian_nb_ext/main.js

```diff
@@ -591,9 +607,10 @@ define([
               $($("input[name=create_opt")[1]).prop("checked", true);
               $("#env_opt option:contains('conda')").prop("selected", true);
             } else {
-              $("#nb_filename_box").val(
-                jvn_params.nb_filename.replace(".ipynb", "")
-              );
+              $("#artifacts_box").val(jvn_params.artifacts);
+              $("#files_box").val(jvn_params.files);
+              $("#notebook_id_box").val(jvn_params.notebook_id);
+              $("#notebook_id_box").val(jvn_params.notebook_id);
```

Comment:
You can remove one, these two are duplicate lines.

Reply:
Sure I will change :)